source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -121,3 +121,50 @@
 		}
 	}
 }
+
+.is-style-features {
+	--wporg--style--feature--offset: 0.2em;
+	padding: 0;
+	list-style: none;
+	font-size: var(--wp--preset--font-size--heading-1);
+	font-weight: 200;
+	line-height: 1.1;
```

Comment:
Variablize this? It's reused on line 168

Reply:
Good idea - it's actually used 3 times, because the `height: 1.1em` is meant to be the same value too. Done in https://github.com/WordPress/wporg-parent-2021/pull/28/commits/479abdc78f45062cab882b8c66f6a1e74f37fce6